boards/pimoroni-tufty2040/README.md

```diff
@@ -0,0 +1,96 @@
+# [pimoroni-tufty2040] - Board Support for the [Pimoroni Tufty2040]
+
+You should include this crate if you are writing code that you want to run on
+a [Pimoroni Tufty2040] - a hackable, programmable badge with a LCD colour
+display, powered by a Raspberry Pi RP2040.
+
+This crate includes the [rp2040-hal], but also configures each pin of the
+RP2040 chip according to how it is connected up on the Tufty2040.
+
+[Pimoroni Tufty2040]: https://shop.pimoroni.com/products/tufty-2040/
+[pimoroni-tufty2040]: https://github.com/rp-rs/rp-hal-boards/tree/main/boards/pimoroni-tufty2040
+[rp2040-hal]: https://github.com/rp-rs/rp-hal/tree/main/rp2040-hal
+[Raspberry Silicon RP2040]: https://www.raspberrypi.org/products/rp2040/
+
+## Using
+
+To use this crate, your `Cargo.toml` file should contain:
+
+```toml
+pimoroni_tufty2040 = "0.1.0"
+```
+
+In your program, you will need to call `pimoroni_tufty2040::Board::take().unwrap()` to create
+a new `Boards` structure. This will set up all the GPIOs for any on-board
+devices and configure common clocks. See the [examples](./examples) folder for more details.
+
+## Examples
+
+### General Instructions
+
+To compile an example, clone the _rp-hal-boards_ repository and run:
+
+```console
+rp-hal-boards/boards/pimoroni-tufty2040 $ cargo build --release --example <name>
+```
+
+You will get an ELF file called
+`./target/thumbv6m-none-eabi/release/examples/<name>`, where the `target`
+folder is located at the top of the _rp-hal-boards_ repository checkout. Normally
+you would also need to specify `--target=thumbv6m-none-eabi` but when
+building examples from this git repository, that is set as the default.
+
+If you want to convert the ELF file to a UF2 and automatically copy it to the
+USB drive exported by the RP2040 bootloader, simply boot your board into
+bootloader mode and run:
+
+```console
+rp-hal-boards/boards/pimoroni-tufty2040 $ cargo run --release --example <name>
+```
+
+If you get an error about not being able to find `elf2uf2-rs`, try:
+
+```console
+$ cargo install elf2uf2-rs
+```
+then try repeating the `cargo run` command above.
+
+### [tufty_blinky](./examples/tufty_blinky)
+
+Flashes the Tufty2040's LED.
```

Comment:
Your example seems to do a lot more than just blink an LED.
If I wrote this, I'd have named the exampe "tufty_demo", and used the description: 
"Draws a circle on the LCD and flashes Tufty2040's LED"